sequelizeDatabase/sequelFunctions.js

```diff
@@ -0,0 +1,13 @@
+const models = require('./modelSetup');
+
+//This function inserts an individual into the Individuals Table.
+insertPerson = (IndividualName, OriginalPosition, LinkedInUrl, Comments) => {
```

Comment:
You are just wrapping a built-in method in-which you have removed functionality.  

Wouldn't it be better for you to call this model creation directly from your route handler?

Reply:
If you wanted to just wrap the method, I'd suggest you take the options as a function. 
```js
insertPerson = (attributes) => {
 models.Individuals.create(attributes).then((user) => {
        console.log('Individual Created: ',user);
    }).catch(err => console.error('Error in insertPerson', err));
} 
```